common/src/stack/storage-config/lib/controller_hpssacli.py

```diff
@@ -227,9 +225,9 @@ def doRaid(self, raidlevel, adapter, enclosure, slots, hotspares,
 
 				cmd = [ 'slot=%d' % adapter, 'array', arrayid,
 					'add', 'spares=%s' % ','.join(spares) ]
-				result = self.run(cmd)
+				result = self.run(cmd, check=check)
 
-	def doGlobalHotSpare(self, adapter, enclosure, hotspares, options):
+	def doGlobalHotSpare(self, adapter, enclosure, hotspares, options, check=True):
```

Comment:
The defaults for `check` seem inconsistent throughout. The top level `run` methods seem to default it to `False`, but the `do*` methods sometimes default it to `True` and other times to `False`. Consider standardizing the defaults, documenting the reasons why the defaults are inconsistent, or getting rid of the defaults altogether in favor of letting the caller pass in `check` and having it filter down.